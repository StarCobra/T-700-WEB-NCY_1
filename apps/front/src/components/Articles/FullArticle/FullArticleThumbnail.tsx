import * as React from 'react';
import '../../../style/article.scss'
import articleExample  from "../../../assets/articles/articleExample.jpg"

export default function FullArticleThumbnail() {
  return (
        <div className="articleThumbnail">
          <img src={articleExample} />
        </div>
  )
}
 