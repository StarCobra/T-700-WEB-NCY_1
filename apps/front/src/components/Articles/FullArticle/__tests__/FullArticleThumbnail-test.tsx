import React from "react";
import { act, create } from "react-test-renderer";
import FullArticleThumbnail from "../FullArticleThumbnail";

describe("Components > FullArticle > FullArticleThumbnail", () => {
  const tests = [
    {
      description: "generic",
      props: {},
    },
  ];
  tests.forEach((test) => {
    it(`should render the component ${test.description}`, async () => {
      let root;
      await act(async () => {
        root = create(<FullArticleThumbnail {...test.props} />);
      });
      expect(root).toMatchSnapshot();
    });
  });
});
