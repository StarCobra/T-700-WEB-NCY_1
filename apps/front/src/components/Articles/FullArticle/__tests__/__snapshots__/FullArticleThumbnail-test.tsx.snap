// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > FullArticle > FullArticleThumbnail should render the component generic 1`] = `
<div
  className="articleThumbnail"
>
  <img
    alt=""
    src="default"
  />
</div>
`;
